### ISSUE TITLE
[Request for Comment] Javascript for the metismenu is neither transpiled nor minified

### ISSUE THREAD

**Issue by richard67** [joomla/cassiopeia#212]
### Problem identified

Currently the javascript file [templates/cassiopeia/js/mod_menu/menu-metismenu.js](https://github.com/joomla/cassiopeia/blob/development/templates/cassiopeia/js/mod_menu/menu-metismenu.js) is not minified by any build script.

When renaming it from `menu-metismenu.js` to `menu-metismenu.es6.js` and running `npm run build:js`, it will transpiled and minified, **but**:

In the original menu-metismenu.es6.js, the same comment is added to the top of the file like it is for the transpiled file:

```
/**
* PLEASE DO NOT MODIFY THIS FILE. WORK ON THE ES6 VERSION.
* OTHERWISE YOUR CHANGES WILL BE REPLACED ON THE NEXT BUILD.
**/
```

### Proposed solution

No idea yet.

### Open questions

What is the right way to transpile and minify that js without adding that comment to the source?

### Additional comments

Related issue #213 : If We have to add copyright information, what I assume, then we really should minify the file.

**Comment by brianteeman** [comment]
I guess you can do it the way that the atum/js is done

1. Move the js folder to media/source/template
2. Remove line 38 from build\build-modules-js\compilejs.es6.js as its not needed
3. rename the file to .es6.js

now when you run the build script the js will be minified and transpiled and copied to
media/template

**Comment by brianteeman** [comment]
You could do just point 3 but as you have seen the file will be overwritten when you run the build script

**Comment by richard67** [comment]
@brianteeman Thanks. Will do that tomorrow or so.

**Comment by richard67** [comment]
Will be solved either with PR #217 or PR #221 .
